charts/spire/charts/spiffe-oidc-discovery-provider/templates/tests/test-connection.yaml

```diff
@@ -33,7 +33,7 @@ spec:
     - name: curl-ingress
       image: {{ template "spire-lib.image" (dict "image" .Values.tests.bash.image "global" .Values.global) }}
       command: ['curl']
-      args: ['-s', '-f', 'http://{{ index .Values.config.domains 0 }}/.well-known/openid-configuration']
+      args: ['-s', '-f', 'http://{{ (index .Values.ingress.hosts 0).host }}/.well-known/openid-configuration']
```

Comment:
Prevents issues when this order is defined differently.

https://github.com/spiffe/helm-charts/pull/324/files#diff-dd133a69512e088172a3954ce5dc656ee751f7871c7d3bb839ca0f6a2675f897R29-R31

This resolved the failing test which I struggled with for a while.